lib/jazzy/config.rb

```diff
@@ -246,6 +246,11 @@ def expand_path(path)
                    'https://github.com/realm/realm-cocoa/tree/v0.87.1)'
 
     # ──────── Doc generation options ────────
+    config_attr :unsearchable,
```

Comment:
I think this should be named either `disable_search`/`--disable-search` or `skip_search`/`--skip-search` to match the other options.